test_scripts/Defects/6_2/1384/common.lua

```diff
@@ -0,0 +1,79 @@
+---------------------------------------------------------------------------------------------------
+-- Common module
+---------------------------------------------------------------------------------------------------
+
+--[[ General configuration parameters ]]
+config.defaultProtocolVersion = 2
+
+--[[ Required Shared libraries ]]
+local actions = require("user_modules/sequences/actions")
+local json = require("modules/json")
+local utils = require("user_modules/utils")
+local runner = require('user_modules/script_runner')
+local hmi_values = require("user_modules/hmi_values")
+local commonRC = require('test_scripts/RC/commonRC')
+
+--[[ Test Configuration ]]
+runner.testSettings.isSelfIncluded = false
+
+--[[ Local Variables ]]
+local m = {}
+
+--[[ Shared Functions ]]
+m.Title = runner.Title
+m.Step = runner.Step
+m.startOrigin = actions.start
+m.preconditions = actions.preconditions
+m.postconditions = actions.postconditions
```

Comment:
@HSavynetska , please remove this link, `postconditions` function used from `actions` file.